src/tiled/tiledproxystyle.cpp

```diff
@@ -123,7 +123,7 @@ static void qt_fusion_draw_arrow(Qt::ArrowType type, QPainter *painter, const QS
     QString cacheKey = uniqueName(QLatin1String("fusion-arrow"), option, rect.size())
             % HexString<uint>(type)
             % HexString<uint>(color.rgba());
-    if (!QPixmapCache::find(cacheKey, cachePixmap)) {
+    if (!QPixmapCache::find(cacheKey, &cachePixmap)) {
```

Comment:
Same here... already [obsolete in Qt 5.6](https://doc.qt.io/archives/qt-5.6/qpixmapcache-obsolete.html#find-1), but still working even with `QT_DISABLE_DEPRECATED_BEFORE=0x050900`.

Btw, I think this change is a bit sad. If you pass in a variable by writable reference, Qt Creator will put it in italic so you can clearly see it may get modified. The `&` was originally meant to make this clearer when we were all coding in Vim or Emacs, but these days I think it makes it less clear. Anyway, enough shedding of tears.

Reply:
Is it worth reporting any of these as issues? At least the `QT_DISABLE_DEPRECATED_BEFORE=0x050900` issue?

Reply:
I guess it's anyway not a good idea to remove functions "retroactively". Since I made my application compile with `QT_DISABLE_DEPRECATED_BEFORE=0x050900`, I would not expect a function that was already deprecated in 5.6 without being wrapped in `QT_DEPRECATED_SINCE` to suddenly disappear when compiling against Qt 5.13. I see they did the expected thing in Qt 5.13, and actually disabled those functions from 5.13 onwards:

```c++
#if QT_DEPRECATED_SINCE(5, 13)
    QT_DEPRECATED_X("Use bool find(const QString &, QPixmap *) instead")
    static QPixmap *find(const QString &key);
    QT_DEPRECATED_X("Use bool find(const QString &, QPixmap *) instead")
    static bool find(const QString &key, QPixmap &pixmap);
#endif
```